webrender/build.rs

```diff
@@ -563,14 +563,14 @@ fn create_vertex_buffer_descriptors(file_name: &str) -> Vec<VertexBufferDesc> {
     } else if file_name.starts_with("debug_color") {
         descriptors = vec![
             VertexBufferDesc {
-                stride: 16, // size of DebugColorVertex 4 * 4
+                stride: 28, // size of DebugColorVertex 3 * 4 + 4 * 4
```

Comment:
let's just use `mem::size_of` directly here?

Reply:
Unfortunately in the build script we can't access structures from the crate itself.

Reply:
We can have those structs declared in a separate module, which is naturally included in the crate but manually included in the build.rs script, e.g.
```rust
#[path = "src/vertex_types.rs"]
mod vertex_types;
```